my-vocabulary/src/functions/parseGPT.js

```diff
@@ -0,0 +1,15 @@
+export const parseGPT = (text) => {
```

Comment:
I would call this method parseChatGptResponse instead.

Reply:
It is not chat gpt though.. I could add the response